libdevice/itt_cmplr_wrappers.cpp

```diff
@@ -0,0 +1,60 @@
+//==--- itt_cmplr_wrappers.cpp - compiler wtappers for ITT -----------------==//
```

Comment:
```suggestion
//==--- itt_cmplr_wrappers.cpp - compiler wrappers for ITT -----------------==//
```

Reply:
I suggest dropping `_cmplr` from the file name: itt_cmplr_wrappers.cpp -> itt_wrappers.cpp

Reply:
We will have `itt_user_wrappers.cpp` later, so I would like to keep this distinction.

Reply:
> We will have `itt_user_wrappers.cpp` later, so I would like to keep this distinction.

Okay. I think we can afford using vowels in the `compiler` word spelling: itt_cmplr_wrappers.cpp -> itt_compiler_wrappers.cpp

Reply:
Sounds reasonable to me :)  Thanks!